### ISSUE TITLE
the app is missing the tests

### ISSUE THREAD

**Issue by pdamoc** [rtfeldman/elm-spa-example#21]
To my understanding, the waited `elm-test` mentioned in `tests/README.md` has been released. 
Even a small example of how to do tests would be useful. 

A more complex example that integrates with `travis-ci` would be even more helpful.

**Comment by aislanmaia** [comment]
I have a impression that the author is waiting for `0.19` :thinking: to add the missing parts

**Comment by rtfeldman** [comment]
Nope, just haven't circled back to this code base in a long time. 😅

**Comment by rtfeldman** [comment]
To set clear expectations, I'm looking at my book writing, OSS projects, and speaking schedule for 2018, and I'm oversubscribed on those alone, even if I did nothing else with my free time this year. With that in mind, the most time I'm going to be able to devote to this repo in 2018 is to update it for new Elm language releases (e.g. 0.19) and do quick fixes for some of the bugs.

I'd love to have bandwidth to add some `elm-test` and especially `elm-html-test` tests for this, but I'm choosing to prioritize writing, OSS, and speaking over that in 2018, which means I'll have bandwidth to add tests in 2019 at the earliest.

**Comment by dwayne** [comment]
@pdamoc @MichaelCombs28 @aislanmaia My implementation has both [unit and fuzz tests](https://github.com/dwayne/elm-conduit/tree/69b1cf41d38e34c4bd8b61490d28e224a0481ac2/tests/Test/Data).
